DESCRIPTION

```diff
@@ -59,5 +59,3 @@ LazyData: true
 RoxygenNote: 7.1.1.9000
 Roxygen: list(markdown = TRUE)
 RdMacros: lifecycle
-Remotes:
```

Comment:
You may want to pin to the new version `generics (>= 0.1.0)`

Reply:
✅